modules/migration-debugging-velero-resources.adoc

```diff
@@ -61,6 +61,23 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
   backup describe 0e44ae00-5dc3-11eb-9ca8-df7e5254778b-2d8ql
 ----
 
+The following types of restore errors and warnings are shown in the output of a `velero describe` request:
+
+* `Velero`: A list of messages related to the operation of Velero itself, for example, messages related to connecting to the cloud, reading a backup file, and so on
+* `Cluster`: A list of messages related to backing up or restoring cluster-scoped resources
+* `Namespaces`: A list of list of messages related to backing up or restoring resources stored in namespaces
+
+One or more errors in one of these categories results in a `Restore` operation receiving the status of `PartiallyFailed` and not `Completed`. Warnings do not lead to a change in the completion status.
+
+[IMPORTANT]
+====
+* For resource-specific errors, that is, `Cluster` and `Namespaces` errors, the `restore describe --details` output includes a resource list that lists all resources that Velero succeeded in restoring. For any resource that has such an error, check to see if the resource is actually in the cluster.
+
+* If there are `Velero` errors, but no resource-specific errors, in the output of a `describe` command, it is possible that the restore completed without any actual problems in restoring workloads, but carefully validate post-restore applications.
++
+For example, if the output contains `PodVolumeRestore` or node sgent-related errors, check the status of `PodVolumeRestores` and `DataDownloads`. If none of these are failed or still running, then volume data might have been fully restored.
```

Comment:
@RichardHoch can you fix this typo in a follow up PR please?

Reply:
On it.